### DISCUSSION TITLE
Forks - mention your works here

### DISCUSSION THREAD

**Discussion by GFW-knocker** [GFW-knocker/gfw_resist_tls_proxy#73] [General]
please mention your good forks and related works
if they get likes we add it to main page
don't mess here by asking irrelevant question
just paste a link and a line for description 

لطفا فورک ها و کارهای خوب خودتون را اینجا بگذارید
اگر لایک بخوره ما در صفحه اصلی قرار میدیم
اینجا رو با سوال های نامرتبط شلوغ نکنید
فقط لینک و توضیح یکی دو خطی

**Comment by Ali-Frh** [comment]
Hi, Thanks for great works here.
i made a gui for this project, check it out in [Here](https://github.com/Ali-Frh/CFuzzy)
i appreciate any comments and suggestion to make it better

**Reply by ghost** [reply]
سلام و ممنون از کار هوشمندانه شما.
روش شما دوباره روش های دور زدن DPI رو زنده کرد و اینجا هم این روش توسط کلاینت PowerTunnel که فقط گزینه --native-frag مخزن GoodbyeDPI رو ایمپلمنت کرده، روی کانفیگ ws پیاده شده.
برای استفاده از کلاینت ویندوزی باید حتما جاوا نصب باشه و پلاگین ها به دایرکتوری اضافه بشن.
با آدرس کثیف discord.com تست شده و بنظر روی همه ISP ها سرعت قابل قبولی داره بجز ایرانسل که خیلی کنده و شما هم اشاره کرده بودید.
سرعت ایرانسل با DoH سرویس Yandex بهتر میشه ولی باز هم کنده. در این صورت باید گزینه Strict DNS Mode هم فعال بشه.
احتمالا اگه بشه از تمامی گزینه های GoodbyeDPI استفاده کرد، نتیجه بهتری گرفت.
البته تا جایی که من متوجه شدم، نه PowerTunnel و نه GoodbyeDPI گزینه 47 قسمت نامساوی و تاخیر کم رو ندارن و روش خودشونو دارن. شاید بشه با یه پلاگین جاوا اضافه کرد.
دمتون گرم

[Telegram Link - PowerTunnel Tutorial](https://t.me/ghafandar/128)
[PowerTunnel Client - Android](https://github.com/krlvm/PowerTunnel-Android)
[PowerTunnel Client - Windows, MacOS, Linux](https://github.com/krlvm/PowerTunnel)
[PowerTunnel Plugin - LibertyTunnel](https://github.com/krlvm/LibertyTunnel)
[PowerTunnel Plugin - DNS](https://github.com/krlvm/PowerTunnel-DNS)
[GoodbyeDPI](https://github.com/ValdikSS/GoodbyeDPI)

**Comment by T-256** [comment]


**Reply by GFW-knocker** [reply]
where is your prj?
404 returned

**Comment by sambali9** [comment]
I have added it to xray-core: https://github.com/sambali9/Xray-core
Usage:
```
{
    "protocol": "freedom",
    "settings": {
        "fragment": {
            "delay": 100, // Delay after each fragment
            "divider": 10, // Number of packets to send, if the original packet size is not divisible by the divider the last packet will be fragment + remainder
            "packetNumber": 1 // nth packet to fragment, if 0 all outbound packets will be fragmented
        }
    }
}
```

**Reply by GFW-knocker** [reply]
thank you for your great work. 
please add "clean usage instruction" into readme of your prj.

**Comment by ghost** [comment]
https://github.com/yebekhe/DPI-TLS-Tunnel
android GUI based on LoadBalancer branch

**Reply by Arya2010** [reply]
این APK موقع نصب Crash میکنه 
نسخه ویندوز نداره ؟

**Reply by ghost** [reply]
فعلا نسخه پایداری نیست
دارم روش کار میکنم

**Comment by Ehsanfarahi22** [comment]
https://github.com/Ehsanfarahi22/Bypassing-SNI-based-HTTPS-Filtering
<div dir="auto">
تغییرات این فورک که امیدوارم به پروژه شما کمک کنند:<br>
1. بسته client hello فقط از وسط SNI فرگمنت میشود و همین کافی است.<br>
2. آیپی های مربوط به یوتیوب، مستقیما از طریق پینگ www.google.com دریافت می شوند که در بهبود سرعت خیلی موثر بود.<br>
3. با فرگمنت کردن، استفاده از سرویس DoH کلادفلر ممکن شده است</div>

**Reply by GFW-knocker** [reply]
خیلی عالی.
مخصوصا اون جمله که گفتید server hello  رو از سمت سرور بشه فرگمنت کرد واقعا عالی میشه
و در اولین فرصت امتحان میکنیم

**Comment by gfwsidehustle** [comment]
درود
برای اسکریپت تیزافزار یوتیوب یه ویدیو ساختم برای سادگی کار دوستان
https://www.youtube.com/watch?v=EhegyoV3LOE
